.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "ethereum/solidity/lib/ds-test"]
 	path = ethereum/solidity/lib/ds-test
 	url = https://github.com/dapphub/ds-test
+[submodule "ethereum/solidity/lib/openzeppelin-contracts"]
```

Comment:
I wonder how were we able to run tests before without having this set here :D

Reply:
Ah, so `forge build` doesn't actually error if it can't find an import... `forge test` does, but given that the QGB contract itself was not tested prior to this PR, then of course there would be no error.

This is of course one of the many examples to Solidity tooling being extremely immature. In the future we should examine using [Sway](https://github.com/FuelLabs/sway) and relaying directly to a Fuel execution layer.